### PR TITLE
(FM-8099) Update README to reflect status of types for XE devices

### PR DESCRIPTION
**Pull request by willmeek**

### CODE REVIEW

**Comment by willmeek** [comment]
@clairecadman We will be looking to make an interim release soon, if the documentation changes here, and the overall README look ok?

For Phase 2 we will probably split out the compatibility table into a separate document.

File: README.md [hunk 1]
```diff
@@ -1027,9 +1027,7 @@ Default value: default
 
 ## Limitations
 
-The following devices have been tested against this module — with the type compatibilities listed.
-
-Note that this is *not* an exhaustive list of supported devices, but rather the results found from execution across a cross section of the devices that we use for internal testing.
+We intend for support across the Catalyst family - both legacy IOS and IOS-XE. We have a finite set of physical platforms we perform continual testing against. In the event of errors or missing abstractions, users are requested to provide the error details (if present) via --debug --trace and a copy of the sanitized configuration (both Puppet and Cisco CLI).
```
Comment (clairecadman) [inline README.md]:
```suggestion
We intend for support across the Catalyst family — both legacy IOS and IOS-XE. We continually perform testing against a limited set of physical platforms. If you experience errors or missing abstractions, provide the error details (if present) via --debug --trace and a copy of the sanitized configuration (both Puppet and Cisco CLI).
```
I'm not sure what "We intend for support across the Catalyst family" means, do you support this now or is it something you plan to do in the future? If the latter then maybe don't say anything?

Comment (willmeek) [reply README.md]: Claire, this would be current functionality - IOS and IOS-XE are both catalyst.

Comment (clairecadman) [reply README.md]: Ah okay, maybe just "We support across the catalyst family - both legacy IOS and IOS-XE" then?  "Intend" makes it sound like you don't currently do this. Also lowercase for 'catalyst'.

Comment (willmeek) [reply README.md]: Changed to

'We support devices across the Catalyst family — both legacy IOS and IOS-XE.'

Comment (willmeek) [reply README.md]: On the bus but will change catalyst to lower case C

I think it was uppercase as it's a type of devices

https://en.m.wikipedia.org/wiki/Cisco_Catalyst

Comment (clairecadman) [reply README.md]: Ah okay, then upper case sounds good!

**Pull request merged**